src/foam/u2/view/OverlayActionListView.js

```diff
@@ -164,25 +188,25 @@ foam.CLASS({
         code: (...rest) => ! rest.reduce((l, r) => l || r, false)
       })));
 
-      this.overlay_.add(this.slot(function() {
-        return this.E().forEach(this.data, function(action) {
-          this.
-            start().
-              show(action.createIsAvailable$(self.__context__, self.obj)).
-              addClass(self.myClass('action')).
-              add(action.label).
-              on('click', function(evt) {
-                self.overlay_.close();
-                action.maybeCall(self.__subContext__, self.obj);
-              }).
-              attrs({
+
+      self.obj.sub(function() {
+        self.overlay_.close();
```

Comment:
didn't delete overlay_ property as it causes slot error

Reply:
Are you still getting the same state error?

Reply:
no, it's something different. that error disappeared after css class name pattern was changed